src/models/mobileservices/mobileservice.js

```diff
@@ -64,14 +66,16 @@ export class MobileService {
     return this.customResourceClass.bindForm(params);
   }
 
-  isBindingOperationInProgress(owner) {
-    const inprogressCR = find(
-      this.customResources,
-      cr => (owner ? find(cr.getOwners(), o => o.name === owner) : true) && cr.isInProgress()
-    );
+  isBindingOperationInProgress(appName) {
+    const inprogressCR = find(this.customResources, cr => this.isOwner(cr, appName) && cr.isInProgress());
     return inprogressCR != null;
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  isOwner(cr, appName) {
```

Comment:
I believe this logic should be part of the [customresource](https://github.com/aerogear/mobile-developer-console/blob/17fcb54ad98a6c051a6e6b88506bb0512f459344/src/models/mobileservices/customresource.js#L11) object by adding there, for example, an `isOwnedBy(appName)`.

That would bring 2 benefits:

1. the method will be automatically available to all the piece of codes that needs to deal with `customresources`
2. the method above could be changed to:
```javascript
const inprogressCR = find(this.customResources, cr => cr.isOwnedBy(appName) && cr.isInProgress());
```
making that more readable.